.github/workflows/build_linux_elf.yml

```
@@ -18,7 +18,8 @@ jobs:
       run: python -m pip install --find-links ${URL} wxPython
     - run: python setup.py clean bundle
     - run: chmod +x /home/runner/work/PythonTurtle/PythonTurtle/dist/PythonTurtle
+    - run: tar -cvf PythonTurtle_for_Linux.tar /home/runner/work/PythonTurtle/PythonTurtle/dist/PythonTurtle
```

With the `z` option in addition yop can turn the result into a GZipped Tar archive, e.g.
```suggestion
    - run: tar -cfvz PythonTurtle_for_Linux.tgz /home/runner/work/PythonTurtle/PythonTurtle/dist/PythonTurtle
```